src/test/java/uk/nhs/digital/nhsconnect/nhais/mapper/PersonNameMapperTest.java

```diff
@@ -42,21 +42,6 @@ void When_MappingPatientName_Then_ExpectCorrectResult() {
         assertEquals(expectedPersonName.toEdifact(), personName.toEdifact());
     }
 
-    @Test
-    public void When_MappingWithoutNhs_Then_IllegalStateExceptionIsThrown() {
```

Comment:
What about a test case where it's null? The thing you've added in `PersonNameMapper` class

Reply:
added test case